src/Mealz/MealBundle/DataFixtures/ORM/LoadDishes.php

```diff
@@ -67,7 +67,7 @@ public function getOrder(): int
     /**
      * add the dishes
      */
-    protected function addDish($titleEN, $titleDE): void
+    protected function addDish($titleEN,$titleDE): void
```

Comment:
We can define type for $titleEN and $titleDE.

Reply:
done.